src/TEvent.h

```diff
@@ -21,6 +22,10 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+#include "pre_guard.h"
+#include <QStringList>
+#include "post_guard.h"
+
```

Comment:
I think the changes here can go away. Or be moved to the `.cpp` file, as needed.